project.html

```diff
@@ -0,0 +1,176 @@
+            <br>
+            <div class="hexagon9">
+                <div class="hexagon-layer-19">
+                    <div class="hexagon-layer9">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon10">
+                <div class="hexagon-layer-110">
+                    <div class="hexagon-layer10">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon11">
+                <div class="hexagon-layer-111">
+                    <div class="hexagon-layer11">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon12">
+                <div class="hexagon-layer-112">
+                    <div class="hexagon-layer12">
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="info">
+            <div class="smit-foto">
+                <div class="smit-foto-layer-1">
+                    <div class="smit-foto-layer">
+                    </div>
+                </div>
+            </div>
+            <div class="smit-line"></div>
+            <div class="smit"></div>
+            <div class="smit-info1"></div>
+            <div class="smit-info2"></div>
+            <div class="badge1"><a href="https://www.facebook.com/campaign/landing.php?&campaign_id=1557999821&extra_1=s%7Cc%7C294723551887%7Cb%7C%D0%B2%D0%BE%D0%B9%D1%82%D0%B8%7C&placement=&creative=294723551887&keyword=%D0%B2%D0%BE%D0%B9%D1%82%D0%B8&partner_id=googlesem&extra_2=campaignid%3D1557999821%26adgroupid%3D58938667677%26matchtype%3Db%26network%3Dg%26source%3Dnotmobile%26search_or_content%3Ds%26device%3Dc%26devicemodel%3D%26adposition%3D1t1%26target%3D%26targetid%3Dkwd-3608757212%26loc_physical_ms%3D1001493%26loc_interest_ms%3D%26feeditemid%3D%26param1%3D%26param2%3D&gclid=CjwKCAiAiJPkBRAuEiwAEDXZZfKb1VSeTwHhdGgikrxVZYJKzhprfF1ClKnCAlc4b99Kbr2IBu8pRxoCiLEQAvD_BwE"><img src="img/Facebook@1X.png" width="24" height="24"></a></div>
+            <div class="badge2"><a href="https://twitter.com/?lang=ru"><img src="img/Twitter@1X.png" width="24" height="24"></a></div>
+            <div class="badge3"><a href="https://www.google.ru/"><img src="img/Google@1X.png" width="24" height="24"></a></div>
+            <div class="badge4"><a href="https://www.pinterest.com/"><img src="img/Pinterest@1X.png" width="24" height="24"></a></div>
+            <div class="badge5"><a href="https://www.linkedin.com/"><img src="img/Linkedin@1X.png" width="24" height="24"></a></div>
+            <div class="badge6"><a href="https://dribbble.com/"><img src="img/Dribbble@1X.png" width="24" height="24"></a></div>
+        </section>
+    </main>
+    <footer>
+        <div class="box-footer"></div>
+        <div class="box-footer-line"></div>
+        <div class="box-footer-line-1"></div>
+
+
+
+        <input class="text-1" type="text" placeholder="    Name (Required)" style="width: 580px; height: 38px;">
```

Comment:
Use `<form></form>`

Reply:
It is input is required, so use attribute `required` for input. As a resut, user doesn't send this form, if he doesn't write your name

Reply:
for creating indent between border of input and text of placeholder, use `padding-left` for input. Don't use spaces in text for it